Source/SpatialGDK/Public/Schema/Rotation.h

```diff
@@ -12,19 +12,19 @@
 
 const Worker_ComponentId ROTATION_COMPONENT_ID = 100001;
 
-struct Rotation : Component
+struct SpatialRotation : SpatialComponent
 {
 	static const Worker_ComponentId ComponentId = ROTATION_COMPONENT_ID;
 
-	Rotation() = default;
+	SpatialRotation() = default;
 
-	Rotation(float InPitch, float InYaw, float InRoll)
+	SpatialRotation(float InPitch, float InYaw, float InRoll)
 		: Pitch(InPitch), Yaw(InYaw), Roll(InRoll) {}
 
-	Rotation(const FRotator& Rotator)
+	SpatialRotation(const FRotator& Rotator)
```

Comment:
Missing include for FRotator

Reply:
It's not missing. The other includes have `FRotator` defined.

Reply:
Via `Platform.h`? Is there another reason to include it? If not I'd replace it with the FRotator include only.

Reply:
Also realise this is tangential to this PR

Reply:
Can change.

Reply:
Yeah I'm glad you realize now several comments in 😡